Makefile

```diff
@@ -6,18 +6,56 @@ clean:
 	find . -name '*~' -delete
 	find . -name '.coverage.*' -delete
 
-PYTEST_CMD := poetry run pytest -s -vv gridstatus/ -n auto --reruns 5 --reruns-delay 3 --durations=25
+PYTEST_CMD := poetry run pytest -s -vv -n auto --reruns 5 --reruns-delay 3 --durations=25
 NOT_SLOW := -m "not slow"
 UNIT_ONLY := -m "not slow and not integration"
 
-.PHONY: test
-test:
-	pip install vcrpy
-	$(PYTEST_CMD) $(NOT_SLOW)
+.PHONY: test-base
+test-base:
+	$(PYTEST_CMD) gridstatus/tests/test_*.py --ignore=gridstatus/tests/source_specific/
+
+.PHONY: test-caiso
+test-caiso:
+	$(PYTEST_CMD) gridstatus/tests/source_specific/test_caiso.py
+
+.PHONY: test-ercot
+test-ercot:
+	$(PYTEST_CMD) gridstatus/tests/source_specific/test_ercot.py gridstatus/tests/source_specific/test_ercot_api.py
+
+.PHONY: test-isone
+test-isone:
```

Comment:
Where is `test_isone_api.py` being run?

Reply:
oop good call, I was thinking both the flatfile and API methods were in the same file. Added.